tests/pytests/unit/utils/test_listdiffer.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+
+from salt.utils import dictdiffer
+from salt.utils.listdiffer import list_diff
+
+
+@pytest.fixture
+def get_NONE_value():
+    return dictdiffer.RecursiveDictDiffer.NONE_VALUE
+
+
+@pytest.fixture
+def get_old_list():
+    return [
+        {"key": 1, "value": "foo1", "int_value": 101},
+        {"key": 2, "value": "foo2", "int_value": 102},
+        {"key": 3, "value": "foo3", "int_value": 103},
+    ]
+
+
+@pytest.fixture
+def get_new_list():
+    return [
+        {"key": 1, "value": "foo1", "int_value": 101},
+        {"key": 2, "value": "foo2", "int_value": 112},
+        {"key": 5, "value": "foo5", "int_value": 105},
+    ]
+
+
+@pytest.fixture
+def test_list_diff(get_old_list, get_new_list):
+    test_list_diff = list_diff(get_old_list, get_new_list, key="key")
+    return test_list_diff
```

Comment:
```suggestion
@pytest.fixture
def list_diff(get_old_list, get_new_list):
    return salt.utils.listdiffer.list_diff(get_old_list, get_new_list, key="key")
```
Try not to name fixtures like `test_*` and `test_` prefixed function is immediately associated to a test case.

Reply:
fixed that, what the heck was I thinking :laughing: